packages/camera/camera_avfoundation/ios/Classes/CameraPlugin.m

```diff
@@ -261,6 +261,7 @@ - (void)handleMethodCallAsync:(FlutterMethodCall *)call
 - (void)handleCreateMethodCall:(FlutterMethodCall *)call
                         result:(FLTThreadSafeFlutterResult *)result {
   // Create FLTCam only if granted camera access (and audio access if audio is enabled)
+  __weak typeof(self) weakSelf = self;
   FLTRequestCameraPermissionWithCompletionHandler(^(FlutterError *error) {
     if (error) {
       [result sendFlutterError:error];
```

Comment:
If self is `nil` this shouldn't be sending error messages or it will hit the same crash.  It also shouldn't continue on to schedule `FLTRequestAudioPermissionWithCompletionHandler`s.  I recommend the `strongSelf`/return early pattern for all of the non-trivial blocks.